lib/si-events-rs/src/audit_log/v1.rs

```diff
@@ -37,34 +37,151 @@ pub enum AuditLogKindV1 {
     ApproveChangeSetApply {
         from_status: ChangeSetStatus,
     },
+    AttachActionFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+        component_id: Option<ComponentId>,
+        action_kind: Option<ActionKind>,
+    },
+    AttachAttributeFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+        component_id: Option<ComponentId>,
+        subject_name: String,
+        prop_id: Option<PropId>,
+        output_socket_id: Option<OutputSocketId>,
+        destination_name: String,
+    },
+    AttachAuthFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+    },
+    AttachCodeGenFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+        component_id: Option<ComponentId>,
+        subject_name: String,
+    },
+    AttachManagementFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+        component_id: Option<ComponentId>,
+        subject_name: String,
+    },
+    AttachQualificationFunc {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        schema_variant_id: Option<SchemaVariantId>,
+        component_id: Option<ComponentId>,
+        subject_name: String,
+    },
     CancelAction {
         prototype_id: ActionPrototypeId,
         action_kind: ActionKind,
         func_id: FuncId,
         func_display_name: Option<String>,
         func_name: String,
     },
+    ContributeModule {
+        version: String,
+        schema_id: Option<SchemaId>,
+        schema_variant_id: Option<SchemaVariantId>,
+        schema_variant_version: Option<String>,
+    },
     CreateChangeSet,
     CreateComponent {
         name: String,
         component_id: ComponentId,
         schema_variant_id: SchemaVariantId,
         schema_variant_name: String,
     },
+    CreateConnection {
+        from_component_id: ComponentId,
+        from_component_name: String,
+        from_socket_id: OutputSocketId,
+        from_socket_name: String,
+        to_component_id: ComponentId,
+        to_component_name: String,
+        to_socket_id: InputSocketId,
+        to_socket_name: String,
+    },
+    CreateFunc {
+        func_display_name: Option<String>,
+        func_kind: FuncKind,
+    },
+    CreateFuncArgument {
+        func_id: FuncId,
+        func_display_name: Option<String>,
+        func_name: String,
+        // strings for now I suppose
+        kind: String,
```

Comment:
Why? 😄

Reply:
oh - this was from we discussed adding types to si-events that are only for audit events. I had this in there before that convo and never came back. I will update!

Reply:
Ah gotcha. I remember this. Sounds good!